packages/web/components/DigestEmailConfigurationFormField/DigestEmailConfigurationFormField.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import {
+  useAddUserInterestMutation,
+  DigestEmailConfiguration as DigestEmailConfigurationType,
+} from '@/generated/graphql'
+import Button, { ButtonVariant } from '@/components/Button'
+import { useTranslation } from '@/config/i18n'
+import Select from '../Select'
+
+type Props = {
+  digestEmailConfig: DigestEmailConfigurationType
+  refetch: () => void
+}
+
+const DigestEmailConfigurationFormField: React.FC<Props> = ({ digestEmailConfig, refetch }) => {
+  const { t } = useTranslation('settings')
+
+  // The filter operation here is just to remove the "WEEKLY" option
+  // until we build support for it
+  const options = (Object.keys(DigestEmailConfigurationType) as (keyof typeof DigestEmailConfigurationType)[]).map(
+    (key, index) => ({
+      value: index,
+      displayName: DigestEmailConfigurationType[key],
+    }),
+  ).filter((option) => option.displayName !== DigestEmailConfigurationType.Weekly);
+
+  // TODO: Figure out why this is -1 even though loggig everything before the OR returns 0
+  const currentConfig = options.find((option) => option.displayName === digestEmailConfig)?.value || -1
```

Comment:
```suggestion
  const currentConfig = options.find((option) => option.displayName === digestEmailConfig)?.value ?? -1
```

Since `0` is falsy `0 || -1` will evaluate to `-1`.  So using the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) should fix this

Reply:
Ahhh yes of course, thanks!